### PR TITLE
New method UnspentOutputs

### PR DESCRIPTION
**Pull request by Kifen**
Added function "UnspentOutputs" which was adapted from a similar function on branch dcrlibwallet-wip

### CODE REVIEW

File: accounts.go [hunk 1]
```diff
@@ -2,6 +2,9 @@ package dcrlibwallet
 
 import (
 	"encoding/json"
+	"fmt"
+	"github.com/decred/dcrd/dcrutil"
```
Comment (itswisdomagain) [inline accounts.go]: Re-arrange your imports.

Comment (Kifen) [reply accounts.go]: Imports re-arranged

File: accounts.go [hunk 2]
```diff
@@ -75,6 +78,44 @@ func (lw *LibWallet) SpendableForAccount(account int32, requiredConfirmations in
 	return int64(bals.Spendable), nil
 }
 
+func (lw *LibWallet) UnspentOutputs(account uint32, requiredConfirmations int32, targetAmount int64) ([]*UnspentOutput, error) {
```
Comment (itswisdomagain) [inline accounts.go]: Take off this `targetAmount` parameter. Use an hardcoded value of 0 in line 86.

File: accounts.go [hunk 3]
```diff
@@ -75,6 +78,44 @@ func (lw *LibWallet) SpendableForAccount(account int32, requiredConfirmations in
...
+	policy := wallet.OutputSelectionPolicy{
+		Account:               account,
+		RequiredConfirmations: requiredConfirmations,
+	}
+	inputDetail, err := lw.wallet.SelectInputs(dcrutil.Amount(targetAmount), policy)
+	// Do not return errors to caller when there was insufficient spendable
+	// outputs available for the target amount.
+	if err != nil && !errors.Is(errors.InsufficientBalance, err) {
```
Comment (itswisdomagain) [inline accounts.go]: This should become `if err != nil {`. Also take off the comments in lines 87 to 88.

**Pull request closed without merging**